### ISSUE TITLE
Turning a SvelteKit project into a Monorepo

### ISSUE THREAD

**Issue by sw-yx** [sw-yx/swyxkit#104]
---
cover_image: https://user-images.githubusercontent.com/6764957/187062740-f908ebd6-66d2-43c0-a84c-0ea5b330e153.png
---

I have written about [why I think most setups should come with monorepo tooling out of the box](https://www.swyx.io/turborepo-why), and it is time to turn swyxkit into a monorepo as well.

This is not an entirely "free" proposition - there is a learning curve, and the tooling overhead must be low for swyxkit to remain useful to a wide audience. However I think the combination of `pnpm` and `turborepo` and the "new Storybook" solutions ( #21 ) mean it is worth adding now. 

![image](https://user-images.githubusercontent.com/6764957/187062740-f908ebd6-66d2-43c0-a84c-0ea5b330e153.png)

## TLDR

I was originally going to blog my work but there were just wayy too many steps. 

Sooooo.. just clone my new boilerplate lol https://github.com/sw-yx/sveltekit-monorepo
